bounties/npm/systeminformation/1/vulnerability.json

```diff
@@ -0,0 +1,54 @@
+{
+  "PackageVulnerabilityID": "1",
+  "DisclosureDate": "2121-02-11",
+  "AffectedVersionRange": "*",
+  "Summary": "Denial of Service",
+  "Contributor": {
+    "Discloser": "Renan Rocha (EffectRenan)",
+    "Fixer": ""
+  },
+  "Package": {
+    "Registry": "npm",
+    "Name": "systeminformation",
+    "URL": "https://www.npmjs.com/package/systeminformation",
+    "Downloads": "792414"
+  },
+  "CWEs": [
+    {
+      "ID": "400",
+      "Description": "The software does not properly control the allocation and maintenance of a limited resource, thereby enabling an actor to influence the amount of resources consumed, eventually leading to the exhaustion of available resources."
+    }
+  ],
+  "CVSS": {
+    "Version": "3",
+    "AV": "N",
+    "AC": "L",
+    "PR": "N",
+    "UI": "N",
+    "S": "U",
+    "C": "N",
+    "I": "N",
+    "A": "L",
+    "E": "X",
+    "RL": "X",
+    "RC": "X",
+    "Score": "5.3"
+  },
+  "CVEs": [""], 
+  "Repository": {
+    "URL": "https://github.com/sebhildebrandt/systeminformation",
+    "Codebase": ["Javascript"],
+    "Owner": "sebhildebrandt",
+    "Name": "Sebastian Hildebrandt",
```

Comment:
Should be `systeminformation`?

Reply:
This should fix up the checks and get your PR auto-merged onto the platform - thanks! 🍰